modules/gdscript/doc_classes/GDScriptFunctionState.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="GDScriptFunctionState" inherits="RefCounted" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
+	<brief_description>
+		A GDScript coroutine.
+	</brief_description>
+	<description>
+	</description>
+	<tutorials>
+		<link title="GDScript documentation index">$DOCS_URL/tutorials/scripting/gdscript/index.html</link>
+	</tutorials>
+	<methods>
+		<method name="cancel">
+			<return type="void" />
+			<description>
+				Cancels coroutine and clears stack.
+				[b]Note:[/b] The result value is always normalized and may not match the key value.
+			</description>
+		</method>
+		<method name="is_valid" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="extended_check" type="bool" default="false" />
+			<description>
+				Verifies if the given state is valid.
+			</description>
+		</method>
+		<method name="resume">
+			<return type="Variant" />
+			<param index="0" name="arg" type="Variant" default="null" />
+			<description>
+				Internal method, do not use.
```

Comment:
To be honest, i dont even see a reason as to why its registered in ClassDB

Reply:
Because it's called by the return from `await` and don't think it can be done with other means like callable pointers, it might be possible and in that case this would be more reasonable, but it might not be for legacy reasons